distributions/vllm/build.yaml

```diff
@@ -7,4 +7,4 @@ distribution_spec:
     safety: meta-reference
     agents: meta-reference
     telemetry: meta-reference
-image_type: conda
\ No newline at end of file
+image_type: conda
```

Comment:
Could you rebase and check again? I ran a `pre-commit` in https://github.com/meta-llama/llama-stack/pull/327